src/custom/state/swap/hooks.ts

```diff
@@ -422,10 +428,10 @@ export function queryParametersToSwapState(parsedQs: ParsedQs, defaultInputCurre
 
   return {
     [Field.INPUT]: {
-      currencyId: inputCurrency,
+      currencyId: inputCurrency === '' ? null : inputCurrency ?? null,
```

Comment:
I understand this is coming from the merge but, hum?

Am I reading this wrong or is this equivalent to `inputCurrecy || null` ?

Reply:
sure seems loik it